OpenTabletDriverPlugins/OpenTabletDriverPlugins.csproj

```diff
@@ -1,11 +1,30 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <PackageReference Include="TabletDriverPlugin" Version="0.2.1"/>
+    <Compile Remove="Driver\**" />
+    <EmbeddedResource Remove="Driver\**" />
+    <None Remove="Driver\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Properties\launchSettings.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="HidSharpCore" Version="1.0.0" />
+    <PackageReference Include="TabletDriverPlugin" Version="0.2.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Properties\" />
   </ItemGroup>
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <Exec Command="xcopy &quot;$(OutDir)OpenTabletDriverPlugins.dll&quot; &quot;%25appdata%25\..\Local\OpenTabletDriver\Plugins&quot; /Y&#xD;&#xA;xcopy &quot;$(OutDir)OpenTabletDriverPlugins.deps.json&quot; &quot;%25appdata%25\..\Local\OpenTabletDriver\Plugins&quot; /Y &#xD;&#xA;xcopy &quot;$(OutDir)OpenTabletDriverPlugins.pdb&quot; &quot;%25appdata%25\..\Local\OpenTabletDriver\Plugins&quot; /Y&#xD;&#xA;" />
+  </Target>
+
 </Project>
```

Comment:
We really shouldn't have to add any of these new options except for `HidSharpCore`, and even then we might be able to mitigate that dependency using direct device paths.